### PR TITLE
add an introduction to folders in the OMERO model

### PR DESCRIPTION
**Pull request by mtbc**
Added a little documentation on Folders in line with discussion on https://trello.com/c/ghvhqiO2/34-omero-developer-docs-for-model-changes. Staged at http://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Model/Containers.html.

### CODE REVIEW

File: omero/developers/Model/Containers.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Reflecting the :model_doc:`June 2016 release <schemas/june-2016.html>`
+of the :model_doc:`OME Data Model <developers/model-overview.html>`,
+OMERO 5.3's object model adds a new kind of container, the Folder. In
+many respects they are rather like Datasets: for example, Folders have a
+description, they may be annotated and they may contain Images. However,
+they are different from Datasets in important respects.
+
+Folders may contain:
+
+  - Images
```
Comment (hflynn) [inline omero/developers/Model/Containers.txt]: these bullet points don't need to be indented

**Comment by jburel** [comment]
Link to what's new for dev/users since it is a feature that people have requested for a long time
i.e. allow me many levels

**Comment by mtbc** [comment]
Link *from* "what's new"? (Is now a good time to edit that without conflicts?)

**Comment by jburel** [comment]
Adding the link is more a reminder. I will cross link the PRs

**Comment by hflynn** [comment]
It's https://github.com/openmicroscopy/ome-documentation/pull/1654 which is currently touching the what's new for devs page. You can add a link to this content there once that's merged.

**Comment by hflynn** [comment]
@mtbc the PR touching whats new for devs is merged now if you want to add a link on this PR

**Comment by hflynn** [comment]
This all looks good to me, happy for me to merge @jburel ?

**Comment by jburel** [comment]
looks good thanks

**Pull request merged**